openedx/core/djangoapps/notifications/email/utils.py

```diff
@@ -0,0 +1,63 @@
+"""
+Email Notifications Utils
+"""
+from django.conf import settings
+from lms.djangoapps.branding.api import get_logo_url_for_email
+from .notification_icons import NotificationTypeIcons
+
+
+def create_datetime_string(datetime_instance):
+    return datetime_instance.strftime('%A, %b %d')
+
+
+def get_icon_url_for_notification_type(notification_type):
```

Comment:
do we need this method when we already have the get icon method in `NotificationTypeIcons` class

Reply:
I can remove it. I thought of making all methods available in `utils.py`.